### ISSUE TITLE
Z-Wave bindings no longer get activated

### ISSUE THREAD

**Issue by pz1** [Z-Wave-Me/home-automation#260]
It seems now that the order in which modules are started are no longer determined by the order in which they appear in the list. In my logs I see the bindings script is consistently started some 3 seconds after the start of the Z-wave bindings.

How do I force a different order, so the bindings will work as before?

See [forum](http://forum.z-wave.me/viewtopic.php?f=3424&t=20849&start=20) for more background

**Comment by PoltoS** [comment]
what do you exactly want?

**Comment by pz1** [comment]
In essence I do want my bindings to function again, so my device statuses will be send over UDP to OpenRemote. 

That worked well until recently. I have verified that the UDP channel does work. The only problematic thing I found in the logs was the fact that my bindings code was called **after** the start of Z-Way. 
You and Pofs have stressed that the order must be the other way around. I guess that is still true. I do not know how I can enforce it. Or has this whole bindings thing changed too lately?

**Comment by pz1** [comment]
> what do you exactly want?

What other information do you need from me? Do you want to continue here or from my [last post](http://forum.z-wave.me/viewtopic.php?f=3424&t=20849) on the forum

**Comment by PoltoS** [comment]
If you are using zway.bind to trap new devices creation, add 0x200 flag ("EnumerateExisting") to get this binding invoked on all already present devices too. See ZWaveBinding/index.js for more info

**Comment by pz1** [comment]
> If you are using zway.bind to trap new devices creation

I am **not** creating new devices. I have manually enumerated all device statuses that need to be bound to send an UDP update message. That worked fine in earlier versions, but not in the last v201 RC's. 

Now, after a system reboot or Z-Way restart the js customcode is properly loaded, and starts, but often the bindings don't seem to be instantiated. At least there are no status updates sent.. RaspiB+Razberry does better thand Raspi2+UZB1.

(I did find four references to EnumerateExisting in the Zwave module. However I have no clue how/where I should apply that in my existing code.)
